src/main/java/com/netflix/imflibrary/st0429_9/AssetMap.java

```diff
@@ -68,16 +68,35 @@
 @Immutable
 public final class AssetMap
 {
-    private static final String assetMap_schema_path = "/org/smpte_ra/schemas/st0429_9_2007/AM/assetMap_schema.xsd";
-
     private final UUID uuid;
     private final List<Asset> assetList = new ArrayList<>();
     private final List<Asset> packingListAssets = new ArrayList<>();
     private final Map<UUID, URI> uuidToPath = new HashMap<>();
     private static final Logger logger = LoggerFactory.getLogger(AssetMap.class);
-    private final AssetMapType assetMapType;
+    private final JAXBElement assetMapTypeJAXBElement;
     public static final List<String> supportedAssetMapSchemaURIs = Collections.unmodifiableList(new ArrayList<String>(){{ add("http://www.smpte-ra.org/schemas/429-9/2007/AM");}});
 
+    public static final List<AssetMapSchema> supportedAssetMapSchemas = Collections.unmodifiableList
+            (new ArrayList<AssetMapSchema>() {{ add( new AssetMapSchema("/org/smpte_ra/schemas/st0429_9_2007/AM/assetMap_schema.xsd", "org.smpte_ra.schemas.st0429_9_2007.AM"));
+                add( new AssetMapSchema("/org/smpte_ra/schemas/st0429_9_2016/AM/assetMap_schema.xsd", "org.smpte_ra.schemas.st0429_9_2016.AM"));}});
```

Comment:
AFAIK, there exists only one schema for AssetMap? Please let me know if that is not correct